model/AdminHandler.php

```diff
@@ -79,7 +79,7 @@ protected function initStruct() {
             'phone'           => pacol( 1,  $passwordReset, 0,      'text', 'pCreate_mailbox_phone', 'pCreate_mailbox_phone_desc', ''),
             'email_other'     => pacol( 1,  $passwordReset, 0,      'mail', 'pCreate_mailbox_email', 'pCreate_mailbox_email_desc', ''),
             'token'           => pacol( 1,          0,      0,      'text', ''                   , ''                  ),
-            'token_validity'  => pacol( 1,          0,      0,      'ts',   ''                   , ''                  ),
+            'token_validity'  => pacol( 1,          0,      0,      'ts',   ''                   , '', '2000-01-01 00:00:00'),
```

Comment:
Please add some whitespace (spaces, no tabs) so that the new default actually starts in the "column" meant for the default.